easybuild/easyconfigs/p/PCMSolver/PCMSolver-20160205-intel-2016a-Python-2.7.11.eb

```diff
@@ -27,7 +27,8 @@ builddependencies = [
     ('Eigen', '3.2.7'),
 ]
 
-configopts = '-DCMAKE_BUILD_TYPE=Release -DEIGEN3_ROOT=$EBROOTEIGEN -DCMAKE_CXX_FLAGS="$LIBLAPACK_MT -DEIGEN_USE_MKL_ALL"'
+configopts = "-DCMAKE_BUILD_TYPE=Release -DEIGEN3_ROOT=$EBROOTEIGEN "
```

Comment:
I tend to use double quotes for strings that include spaces, unless there's a technical reason not to (e.g. double quotes required in the string itself).

See WIP version of easyconfigs style guide at http://boegel-eb.readthedocs.io/en/easyconfigs_style/Code_style.html#code-style-easyconfigs-string-quotes